hudi-client/hudi-client-common/src/main/java/org/apache/hudi/client/transaction/lock/StorageBasedLockProvider.java

```diff
@@ -125,13 +119,12 @@ private synchronized void setLock(StorageLockFile lockObj) {
    *                          ConditionalWriteLockConfig
    * @param conf              Storage config, ignored.
    */
-  public ConditionalWriteLockProvider(final LockConfiguration lockConfiguration, final StorageConfiguration<?> conf) {
-    ConditionalWriteLockConfig config = new ConditionalWriteLockConfig.Builder()
+  public StorageBasedLockProvider(final LockConfiguration lockConfiguration, final StorageConfiguration<?> conf) {
```

Comment:
The conf is never used?

Reply:
All the ext writer lock providers (zookeeper, dynamo) look like this unfortunately

Reply:
We need it for reflection loading the lock provider impl